docs/ufp.md

```diff
@@ -0,0 +1,25 @@
+Ultimaker Format Package (UFP)
+============
+
+Purpose:
+--------
+The goal of this file type is to provide required data for the 3D printer to perform a print job.
+
+Structure:
+--------
+A UFP file contains the following files:</br>
```

Comment:
In Markdown you can use the following format for a list:

* item 1
* item 2
* item 3
  * subitem